File: kotlin/sprint0/B.kt
private fun zip(a: List<Int>, b: List<Int>): List<Int> {
    TODO()
    // Your code
    // ヽ(´▽`)/
}

fun main() {
    val size = readInt()
    val a = readInts()
    val b = readInts()
    println(zip(a, b).joinToString(" "))
}

private fun readStr() = readln()
private fun readInt() = readStr().toInt()
private fun readStrings() = readStr().split(" ")
private fun readInts() = readStrings().map { it.toInt() }

File: swift/sprint0/B.swift
func zip(_ a: [Int], _ b: [Int]) -> [Int] {
    // Your code
    // “ヽ(´▽｀)ノ”
}

func readInts() -> [Int] {
    return readLine()!.split(separator: " ").map { (x: Substring) -> Int in
        return Int(x)!
    }
}

func printArray(_ a: [Int]) {
    for element in a {
        print(element, terminator: " ")
    }
}

let size = Int(readLine()!)!
let a = readInts()
let b = readInts()
let zipped = zip(a, b)
printArray(zipped)
